content/en/docs/k8s-operator/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: OpenTelemetry Operator for Kubernetes
+linkTitle: K8s Operator
+weight: 11
+description:
+  An implementation of a Kubernetes Operator, that
+  manages collectors and auto-instrumentation of the workload using OpenTelemetry
+  instrumentation libraries.
+spelling: cSpell:ignore Otel
+aliases: [/docs/operator]
+---
+
+## Introduction
+
+The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/).
+
+The operator manages:
+
+- OpenTelemetry Collector
+- [auto-instrumentation of the workloads using OpenTelemetry instrumentation libraries](https://github.com/open-telemetry/opentelemetry-operator#opentelemetry-auto-instrumentation-injection)
+
+## Getting started
+
+To install the operator in an existing cluster, make sure you have cert-manager 
+installed and run:
+
+```bash
+$ kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
+```
+
+Once the `opentelemetry-operator` deployment is ready, create an OpenTelemetry
+ Collector (otelcol) instance, like:
+
+```bash
+$ kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: simplest
+spec:
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+
+    exporters:
+      logging:
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: []
+          exporters: [logging]
+EOF
+```
+
+For more configuration options and for setting up the injetion of
+auto-instrumentation of the workloads using OpenTelemetry instrumentation
+libraries, continue reading [here](https://github.com/open-telemetry/opentelemetry-operator/blob/main/README.md).
+
+## Helm Chart
```

Comment:
I am not sure if this is officially maintained. In the past, the helm chart repo was looking for maintainers.

Reply:
I was not aware of that, good to know, let me remove that section then for now.

Reply:
@dmitryax was maintaining the helm charts in the past. 

@dmitryax is the helm chart repo actively maintained?

Reply:
I removed the helm chart section for now, but let's follow up on that, would be good to know where the helm charts are standing.